<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthEncodingDialog</class>
 <widget class="QDialog" name="DepthEncodingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Format</string>
  </property>
  <widget class="QPushButton" name="start_depth_encoding">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Depth Encoding</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QComboBox" name="depth_source_type">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IMAGE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth Source Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoding Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="depth_encoding_type">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jet Mapping</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Surface Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HHA</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
